api/coreservice.go

```diff
@@ -497,14 +497,23 @@ func (core *coreService) SendAction(ctx context.Context, in *iotextypes.Action)
 		return "", st.Err()
 	}
 	// If there is no error putting into local actpool, broadcast it to the network
-	if core.messageBatcher != nil {
+	// TODO: broadcast action hash if it's blobTx
+	isBlobTx := false
+	out := proto.Message(in)
```

Comment:
why need to add `proto.Message()`

Reply:
for unifying sending action and action hash